.github/workflows/build-test-osx-x86.yaml

```diff
@@ -88,6 +88,8 @@ jobs:
         id: cache-opam
         uses: actions/cache@v3
         if: ${{ inputs.use-cache }}
+        env:
```

Comment:
would be better to put what was in your PR comment in the code itself, so next time someone look at this they can understand the motivations for those values (yes one can use git blame, but better to avoid one indirection).